test.js

```diff
@@ -13,6 +13,10 @@ test('v4 with impossible timeout', async t => {
 	t.false(await isOnline({timeout: 1}));
 });
 
+test('check url', async t => {
+	t.true(await isOnline({url: 'https://google.com'}));
```

Comment:
You're not actually testing the  `url` option here as the IP checks are faster. You need to mock those out.